### ISSUE TITLE
Is there a view for managing permissions?

### ISSUE THREAD

**Issue by felipefrancisco** [Zizaco/entrust#469]
Is there a view for managing permissions? An interface to manage which role has some permissions and etc.

**Comment by jjanusch** [comment]
I don't believe that is included, or at least I haven't seen one. I don't think it would make sense to do one since everyone is going to have different templates, different ways of handling forms, different URL structures, etc. Thankfully, it's not the most difficult thing to build yourself and re-use between projects.

**Comment by felipefrancisco** [comment]
Actually, I was referring to a "Manage Permissions" view, like [this](https://www.aclmanager.net/images/screenshots/aclmanager3.jpg). I don't think it should be included in the default package, but I was wondering if anyone has already made it and open-sourced it.

**Comment by kevinranks** [comment]
@felipefrancisco This would be great and save me a lot of time!

**Comment by poxin13** [comment]
This exists: https://github.com/acoustep/entrust-gui
